python/pynq/drivers/usb_wifi.py

```diff
@@ -43,12 +40,26 @@ class Usb_Wifi(object):
 
     The board is compatible with RALink RT5370 devices.
 
+    Note
+    ----
+    Administrator rights are necessary to create network interface file
     Attributes
```

Comment:
@LucaCerina , I think you need to add a blank line between the actual note and the attributes below it. Otherwise the docstring might be corrupted.